CODE_OF_CONDUCT.md.template

```diff
@@ -0,0 +1,6 @@
+# Code of conduct
+
+By participating in this project, you agree to abide by the
```

Comment:
What does the word "participating" mean in this context? Do we need to define it?

Reply:
Not sure.

In the linked document we say:

> We expect everyone to follow these rules anywhere in thoughtbot's project codebases, issue trackers, chatrooms, and mailing lists.

I'd like to avoid duplicating text from there, though. Is there a phrasing that is more clear?

Reply:
Thinking on this more, it seems to me that in order to violate our code of conduct you would first need to participate in one of our projects. If we've linked to this information in the clearest possible location(s), then I think we're good.